.bundle/config

```diff
@@ -0,0 +1,2 @@
+---
+BUNDLE_BUILD__MYSQL2: "--with-ldflags=-L/usr/local/opt/libmysqlclient/lib --with-cppflags=-I/usr/local/opt/libmysqlclient/include"
```

Comment:
was this file autogenerated?

Reply:
Yes— I'm not too sure how it got there; should I remove this and add it to the .gitignore?

Reply:
Sorry for the late reply! Yep, let's remove and gitignore it